clients/python/.openapi-generator-ignore

```diff
@@ -29,3 +29,4 @@ lakefs_client/client.py
 Gemfile.lock
 Gemfile
 _config.yml
+templates
```

Comment:
```suggestion
templates/
```

Reply:
Hi @arielshaqed ,

It will refer to the README present under clients/python folder. Please ping me over slack if you have any queries